zhusuan/distributions/multivariate.py

```diff
@@ -148,6 +157,102 @@ def _prob(self, given):
         return tf.exp(self._log_prob(given))
 
 
+class UnnormalizedMultinomial(Distribution):
```

Comment:
Maybe we could add an alias like `BagofCategorical`? How about this name? @cjf00000